docs/analytics/what-is-amplitude.md

```diff
@@ -2,145 +2,54 @@
 
-To understand how Amplitude works, let’s walk through a hypothetical example.
+Amplitude is a powerful product analytics platform that enables you to build better products by tracking and understanding user behavior. 
 
-Tunes is a standard music player for mobile devices that has common actions like playing a song, skipping a song, shuffling play, and sharing a song.
+With Amplitude, you can track user data and gain insights into user engagement, retention, and revenue. Amplitude keeps your data trustworthy and secure, ensuring that you have access to accurate and reliable information. Amplitude offers powerful analytics tools that help answer questions about what happened, why it happened, and which actions to take next. With Amplitude, you can seamlessly share your work across teams, facilitating collaboration and driving growth.
 
-Using Amplitude, you can track all the actions your users make in detail and better understand what’s working and what’s not.
+## Key concepts
 
-### What to track
+Amplitude is an event-based analytics tool that tracks the behaviors of users based on in-product interactions and analyzes user behavior in real-time. Event-based analytics is the method of tracking and analyzing interactions between users and your product, also known as events. Before you get started with Amplitude, it's important to know the key concepts used in Amplitude. 
 
-Amplitude gives you the power to decide what actions you want to track. You can choose to track anything and everything.
+|Name|Description|A music player app example|
+|---|-----------|-----------|
+|Users|A user is the specific individual that interacts with your product|A user uses the app to play music|
+|Events|An event is an action a user has taken|A user presses the "Play Song" button|
+|User Properties|User properties are details about users|Whether a user is on a paid or free plan|
+|Event Properties|Event properties are details about events|The genre of the music|
+|Sessions|A session is the period of time a user has your app|A user uses the app to play music|
 
-For example, in Tunes, you could track the music control buttons the users presses or the number of songs each user has listened to in their sessions.
+TL;DR: If you aren't interested in the details of these concepts you can stop reading now. 
 
-#### Events represent actions users have taken
+### Users
```

Comment:
Events section go first.

Reply:
Put Events section before Users. Also nested event properties under events and user properties under users to make closer to each other for better loop up and readability. 

Current structure: 
- Events
   - Event properties 
- Users 
   - User properties 
- Sessions